packages/mdc-select/component.ts

```diff
@@ -165,11 +141,11 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> {
     this.selectedText_!.addEventListener('keydown', this.handleKeydown_);
     this.menu_!.listen(menuSurfaceConstants.strings.CLOSED_EVENT, this.handleMenuClosed_);
     this.menu_!.listen(menuSurfaceConstants.strings.OPENED_EVENT, this.handleMenuOpened_);
-    this.menu_!.listen(menuConstants.strings.SELECTED_EVENT, this.handleMenuSelected_);
-    this.foundation_.setupMenu();
+    this.menu_!.listen(menuConstants.strings.SELECTED_EVENT, this.handleMenuItemAction_);
+    this.foundation_.init();
 
     // Initially sync floating label
-    this.foundation_.handleChange(/* didChange */ false);
+    this.foundation_.updateLabel();
```

Comment:
Can we call this inside foundation's init method and make it a private method of foundation? This foundation method is never called except on init.